test/sql/window/test_parallel_window.test_slow

```diff
@@ -2,24 +2,12 @@
 # description: Hashing and parallel execution
 # group: [window]
 
-#statement ok
-#PRAGMA enable_verification
```

Comment:
Are these changes intentional?

Reply:
Yes I just deleted some commented out lines. There were failures here and I wanted to be sure I knew what was actually being used.